pkg/k8s/config_map_cache_informer.go

```diff
@@ -0,0 +1,132 @@
+package k8s
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/informers"
+	infcorev1 "k8s.io/client-go/informers/core/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+)
+
+type InformerConfigMap struct {
```

Comment:
can we call this `InformerConfigMapUpdater` or something similar with `Updater` in the name? I think such a name will help ensure its purpose is clear.

```suggestion
type InformerConfigMapUpdater struct {
```

Reply:
I agree with this change.